front/lib/models/feature_flag.ts

```diff
@@ -0,0 +1,63 @@
+import type {
+  CreationOptional,
+  ForeignKey,
+  InferAttributes,
+  InferCreationAttributes,
+} from "sequelize";
+import { DataTypes, Model } from "sequelize";
+
+import { front_sequelize } from "@app/lib/databases";
+import { Workspace } from "@app/lib/models/workspace";
+
+export class FeatureFlag extends Model<
+  InferAttributes<FeatureFlag>,
+  InferCreationAttributes<FeatureFlag>
+> {
+  declare id: CreationOptional<number>;
+  declare createdAt: CreationOptional<Date>;
+  declare updatedAt: CreationOptional<Date>;
+
+  declare name: string;
```

Comment:
Let's have a type here?

Reply:
Ok, where would you define it ? `@dust-tt/types` ?

Reply:
Put it in `WhitelistableFeature`

Reply:
(in `types`)